### PR TITLE
[CU-384qkzy] Update docs for Pablo

### PR DESCRIPTION
**Pull request by Martin-Kiss**
## Issue

https://app.clickup.com/t/384qkzy

## Additional Changes

This PR specifically fixes the affected book documentation

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-384qkzy Update docs for pablo to be in line with dex refactor](https://app.clickup.com/t/384qkzy)

File: docs/docs/products/pablo-overview.md [hunk 1]
```diff
@@ -5,17 +5,23 @@ basic need of any DeFi ecosystem. In this case that function is token swaps whil
...
 
-Initially, Pablo will focus on achieving strong liquidity for Substrate-native assets on our Kusama and Polkadot 
-parachains, and then continue to evolve as future infrastructure such as our Substrate < > IBC bridge — Centauri, 
-goes live. Therefore, Pablo is positioned not only to capture liquidity from the DotSama ecosystem but also to become 
-a cross-chain hub that can natively tap into liquidity from IBC and EVM enabled blockchains.
+Initially, 
+Pablo will focus on achieving strong liquidity for Substrate-native assets on our Kusama and Polkadot parachains, 
+and then continue to evolve as future infrastructure such as our Substrate <<!--\-->>IBC bridge — Centauri, goes live. 
+Therefore, Pablo is positioned not only to capture liquidity from the DotSama ecosystem
+but also to become a cross-chain hub that can natively tap into liquidity from IBC and EVM enabled blockchains.
```
Comment (JafarAz) [inline docs/docs/products/pablo-overview.md:23]:
```suggestion
but also to become a cross-chain hub that can natively tap into liquidity from IBC-enabled blockchains.
```

File: docs/docs/products/pablo/cross-chain-DEX.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ interoperable technology stack that integrates with most major blockchain ecosys
 
 Picasso uses the, DotSama XCM channels and the Centauri IBC-Substrate bridge. 
 XCM channels allow Picasso to connect to any parachain with DotSama where Pablo can be leveraged by other projects who
```
Comment (JafarAz) [inline docs/docs/products/pablo/cross-chain-DEX.md:10]:
```suggestion
XCM channels allow Picasso to connect to any parachain in the DotSama ecosystem where Pablo can be leveraged by other projects who 
```

File: docs/docs/products/pablo/cross-chain-DEX.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ interoperable technology stack that integrates with most major blockchain ecosys
 
 Picasso uses the, DotSama XCM channels and the Centauri IBC-Substrate bridge. 
 XCM channels allow Picasso to connect to any parachain with DotSama where Pablo can be leveraged by other projects who 
-would like a DEX for their tokens, bonding opportunities to increase POL and a platform to launch a LBP for launching 
+would like a DEX for their tokens, bonding opportunities to increase POL and a platform to launch an LBP for launching
```
Comment (JafarAz) [inline docs/docs/products/pablo/cross-chain-DEX.md:11]:
```suggestion
wish to launch a token on a DEX through various auction methods, provide liquidity incentives, and bonding opportunities to increase POL.
```

File: docs/docs/products/pablo/cross-chain-DEX.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ interoperable technology stack that integrates with most major blockchain ecosys
 
 Picasso uses the, DotSama XCM channels and the Centauri IBC-Substrate bridge. 
 XCM channels allow Picasso to connect to any parachain with DotSama where Pablo can be leveraged by other projects who 
-would like a DEX for their tokens, bonding opportunities to increase POL and a platform to launch a LBP for launching 
+would like a DEX for their tokens, bonding opportunities to increase POL and a platform to launch an LBP for launching 
 tokens in a fair method whilst finding price discovery.
```
Comment (JafarAz) [inline docs/docs/products/pablo/cross-chain-DEX.md:12]:
```suggestion
```

File: docs/docs/products/pablo/cross-chain-DEX.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ interoperable technology stack that integrates with most major blockchain ecosys
 
 Picasso uses the, DotSama XCM channels and the Centauri IBC-Substrate bridge.
```
Comment (JafarAz) [inline docs/docs/products/pablo/cross-chain-DEX.md:9]:
```suggestion
Picasso uses the DotSama XCM channels and the Centauri IBC-Substrate bridge. 
```

File: docs/docs/products/pablo/cross-chain-DEX.md (hunk 3 above)
Comment (Martin-Kiss) [reply docs/docs/products/pablo/cross-chain-DEX.md:11]: What "various auction methods", please elaborate.

File: docs/docs/products/pablo/swaps-trading.md [hunk 6]
```diff
@@ -4,19 +4,20 @@ Pablo’s primary function is token exchange, to enable users to trade and excha
...
 are all distributed back to liquidity providers (LPers) and PBLO token holders.
 
-On launch Pablo will support Uniswap 50/50 pools, Balancer pools, Stableswap pools, as well as our custom LBP 
-implementation. Stableswap pools allow for superior trading of stablecoins or assets of the same price, providing 
-greater yield to LPs, and less price slippage for traders.  Balancer pools, which are commonly deployed as 80/20 pools, 
-are better suited for high volatility assets and blunt the impact of impermanent loss. In addition, pool creators are 
-able to set fees based on the relative risks of their provided assets, whereas other protocols often set fixed fees at 
-0.3% across all pairs. The volatility and volume can depend significantly depending on the pool assets, and therefore 
-increasing the customizability will boost efficiency of the DEX.
+On launch Pablo will support Balancer pools,
```
Comment (0xBrainjar2) [inline docs/docs/products/pablo/swaps-trading.md:14]: afaik first it is xy=k pools, then the rest of those

**Review by 0xBrainjar2 — APPROVED** [review]
Minor comment then lgtm

File: docs/docs/products/pablo/swaps-trading.md (hunk 6 above)
Comment (Martin-Kiss) [reply docs/docs/products/pablo/swaps-trading.md:14]: @0xBrainjar2 If I am not mistaken, the Balancer pools (constant product pools) are the ones you mention as xy=k pools. @vimukthi-git can you confirm or provide some context?

Comment (vimukthi-git) [reply docs/docs/products/pablo/swaps-trading.md:14]: Confirm! We only support dual-asset-constant-product (xy = k) at first

**Review by KaiserKarel — APPROVED** [review]

File: docs/docs/products/pablo/swaps-trading.md [hunk 7]
```diff
@@ -4,19 +4,22 @@ Pablo’s primary function is token exchange, to enable users to trade and excha
...
 
-On launch Pablo will support Uniswap 50/50 pools, Balancer pools, Stableswap pools, as well as our custom LBP 
-implementation. Stableswap pools allow for superior trading of stablecoins or assets of the same price, providing 
-greater yield to LPs, and less price slippage for traders.  Balancer pools, which are commonly deployed as 80/20 pools, 
-are better suited for high volatility assets and blunt the impact of impermanent loss. In addition, pool creators are 
-able to set fees based on the relative risks of their provided assets, whereas other protocols often set fixed fees at 
-0.3% across all pairs. The volatility and volume can depend significantly depending on the pool assets, and therefore 
-increasing the customizability will boost efficiency of the DEX.
+On launch Pablo will support Constant product pools based on [BalancersFi weighted math], 
+later on followed by Uniswap 50/50 pools, Stableswap pools, as well as our custom LBP implementation.
```
Comment (0xBrainjar2) [inline docs/docs/products/pablo/swaps-trading.md:15]: on launch will support xy=k 

then balancer stableswap, lbp coming later

**Review by 0xBrainjar2 — APPROVED** [review]
Plz resolve comment

File: docs/docs/products/pablo/swaps-trading.md (hunk 7 above)
Comment (vimukthi-git) [reply docs/docs/products/pablo/swaps-trading.md:15]:
```suggestion
later on followed by Balancer based Stableswap pools, as well as LBP implementation. 
```

File: docs/docs/products/pablo/swaps-trading.md [hunk 8]
```diff
@@ -4,19 +4,22 @@ Pablo’s primary function is token exchange, to enable users to trade and excha
...
 are all distributed back to liquidity providers (LPers) and PBLO token holders.
 
-On launch Pablo will support Uniswap 50/50 pools, Balancer pools, Stableswap pools, as well as our custom LBP 
-implementation. Stableswap pools allow for superior trading of stablecoins or assets of the same price, providing 
-greater yield to LPs, and less price slippage for traders.  Balancer pools, which are commonly deployed as 80/20 pools, 
-are better suited for high volatility assets and blunt the impact of impermanent loss. In addition, pool creators are 
-able to set fees based on the relative risks of their provided assets, whereas other protocols often set fixed fees at 
-0.3% across all pairs. The volatility and volume can depend significantly depending on the pool assets, and therefore 
-increasing the customizability will boost efficiency of the DEX.
+On launch Pablo will support Constant product pools based on [BalancersFi weighted math],
```
Comment (vimukthi-git) [inline docs/docs/products/pablo/swaps-trading.md:14]:
```suggestion
On launch Pablo will support dual asset Constant product(x * y = k) pools based on [BalancersFi weighted math], 
```

**Pull request merged**